packages/blade/src/_helpers/storybook/ComponentStatusTable.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import styled from 'styled-components';
+import type { IconComponent } from '../../components/Icons';
+import {
+  AlertCircleIcon,
+  EditIcon,
+  CheckIcon,
+  ClockIcon,
+  LoaderIcon,
+} from '../../components/Icons';
+import { Text } from '~components/Typography';
+import { makeSpace } from '~utils';
+import Box from '~components/Box';
+import type { BadgeProps } from '~components/Badge';
+import { Badge } from '~components/Badge';
+import { Link } from '~components/Link';
+
+type ComponentStatuses =
+  | 'released'
+  | 'in-api-spec'
+  | 'in-development'
+  | 'in-design'
+  | 'deprecated'
+  | `planned-Q${1 | 2 | 3 | 4}`;
+
+type ComponentStatusData = {
+  name: string;
+  status: ComponentStatuses;
+  releasedIn?: string;
+}[];
+
+const componentData: ComponentStatusData = [
+  {
```

Comment:
add data for the components we are doing this quarter

Reply:
Will do.